manifests/tftp/netboot.pp

```diff
@@ -43,12 +43,17 @@
         source => "/boot/efi/EFI/${grub_efi_path}/grubx64.efi",
       }
 
-      file { "${root}/grub2/${shim_file}":
+      file { "${root}/grub2/shimx64.efi":
         ensure => file,
         source => "/boot/efi/EFI/${grub_efi_path}/${shim_file}",
         mode   => '0644',
         owner  => 'root',
       }
+      
```

Comment:
Lint complains about trailing whitespace here.

Reply:
Thanks, added one more empty line in debian section to comply with the rest of the file. Rebased.